Modules/_ssl.c

```diff
@@ -5767,10 +5801,20 @@ _ssl_enum_crls_impl(PyObject *module, const char *store_name)
     Py_XDECREF(enc);
     Py_XDECREF(tup);
 
-    /* CERT_CLOSE_STORE_FORCE_FLAG forces freeing of memory for all contexts
-       associated with the store, in this case our collection store and the
-       associated system stores. */
-    if (!CertCloseStore(hCollectionStore, CERT_CLOSE_STORE_FORCE_FLAG)) {
+    /* When a collection store and its sibling stores are closed
+       with CertCloseStore using CERT_CLOSE_STORE_FORCE_FLAG,
+       the collection store must be closed before its sibling stores.
+      (https://docs.microsoft.com/en-us/windows/win32/api/wincrypt/nf-wincrypt-certaddstoretocollection) */
+    success = CertCloseStore(hCollectionStore, CERT_CLOSE_STORE_FORCE_FLAG);
```

Comment:
You could remove the `BOOL success;` declaration before and do this?:
```suggestion
    BOOL success = CertCloseStore(hCollectionStore, CERT_CLOSE_STORE_FORCE_FLAG);
```